analytics-webapps/src/main/webapp/vue-app/analysis-chart/main.js

```diff
@@ -0,0 +1,32 @@
+import './initComponents.js';
+
+Vue.use(Vuetify);
+const vuetify = new Vuetify({
+  dark: true,
+  iconfont: '',
+});
+
+document.dispatchEvent(new CustomEvent('displayTopBarLoading'));
+
+const appId = 'analysisChart';
+
+// getting language of user
+const lang = eXo && eXo.env && eXo.env.portal && eXo.env.portal.language || 'en';
+const url = `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.addon.Analytics-${lang}.json`;
+
+export function init() {
+  exoi18n.loadLanguageAsync(lang, url).then(i18n => {
+    const appElement = document.createElement('div');
+    appElement.id = appId;
+
+    // init Vue app when locale ressources are ready
+    new Vue({
+      mounted() {
+        document.dispatchEvent(new CustomEvent('hideTopBarLoading'));
+      },
+      template: `<analysis-chart v-cacheable id="${appId}" />`,
```

Comment:
it's better to avoid using skeleton V2 here with `v-cacheable`, we should display only fresh data to user